changelog.md

```diff
@@ -1,3 +1,183 @@
+}
+
+// v3
+const { result: item }  = container.items.read(id, undefined)
+if (item === undefined) { console.log('item not found') }
+```
+
+#### Default Multi Region Write (#335)
+
+The SDK will now write to multiple regions by default if your database configuration supports it. This was previously opt-in behavior.
+
+#### Proper Error Objects (#334, Community Request)
+
+Failed requests now throw proper `Error` or subclasses of `Error`. Previously they threw plain JS objects.
+
+### New Features
+
+#### User Cancellable Requests (#263, Community Request)
+
+The move to `fetch` internally allows us to use the browser `AbortController` API to support user cancellable operations. In the case of operations where multiple requests are potentially in progress (like cross partition queries), all requests for the operation will be canceled. Modern browser users will already have `AbortController`. Node.js users will need to use a [polyfill library](https://www.npmjs.com/package/node-abort-controller)
+
+``` js
+ const controller = new AbortController()
+ const {result: item} = await items.query('SELECT * from c', { abortSignal: controller.signal});
+ controller.abort()
+```
+
+#### Set throughput as part of db/container create operation (#220)
+
+``` js
+    const { database }  = client.databases.create({ id: 'my-database', throughput: 10000 })
+    database.containers.create({ id: 'my-container', throughput: 10000 })
+```
+
+#### @azure/cosmos-sign (#213)
+
+Header token generation was split out into a new library, @azure/cosmos-sign. Anyone calling the Cosmos REST API directly can use this to sign headers using the same code we call inside @azure/cosmos.
+
+#### UUID for generated IDs (#355)
+
+v2 had custom code to generate item IDs. We have switched to the well known and maintained community library `uuid`.
+
+#### Connection Strings (#350, Community Request)
+
+It is now possible to pass a connection string copied from the Azure portal:
+
+``` js
+const client = new CosmosClient("AccountEndpoint=https://test-account.documents.azure.com:443/;AccountKey=c213asdasdefgdfgrtweaYPpgoeCsHbpRTHhxuMsTaw==;")
+```
+
+#### Add DISTINCT and LIMIT/OFFSET queries (#306)
+
+``` js
+ const { results } = await items.query('SELECT DISTINCT VALUE r.name FROM ROOT').fetchAll()
+ const { results } = await items.query('SELECT * FROM root r OFFSET 1 LIMIT 2').fetchAll()
+```
+
+### Improved Browser Experience
+
+While it was possible to use the v2 SDK in the browser it was not an ideal experience. You needed to polyfill several node.js built-in libraries and use a bundler like Webpack or Parcel. The v3 SDK makes the out of the box experience much better for browser users.
+
+- Replace request internals with `fetch` (#245)
+- Remove usage of Buffer (#330)
+- Remove node builtin usage in favor of universal packages/APIs (#328)
+- Switch to node-abort-controller (#294)
+
+### Bug Fixes
+
+a27dfbf Fix offer read and bring back offer tests (#224)
```

Comment:
This doesn't render properly as a list. It gets rendered as a paragraph. You should add `-` or something in the front (or double space between lines)